src/services/loginService.ts

```diff
@@ -32,9 +34,25 @@ export class AzureLoginService {
     }
   }
 
-  public static async interactiveLogin(options: InteractiveLoginOptions): Promise<AuthResponse> {
-    await open("https://microsoft.com/devicelogin");
-    return await interactiveLoginWithAuthResponse(options);
+  public static async interactiveLogin(options?: InteractiveLoginOptions): Promise<AuthResponse> {
+    // await open("https://microsoft.com/devicelogin");
+    // return await interactiveLoginWithAuthResponse(options);
+    var authResp: AuthResponse = {credentials: undefined, subscriptions: []};
+    if(!(options.tokenCache as SimpleFileTokenCache).empty()){
```

Comment:
is `empty()` a required function on the tokenCache? Or just an additional method you created?  If you created I would change to `isEmpty()` since you are returning a boolean expression.